submissions/DmitryHniezdilov/task-html-css-popup/css/style.css

```diff
@@ -0,0 +1,298 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+*:focus-visible {
+  outline: none;
+}
+
+.container {
+  position: static;
+  min-height: 100vh;
+  width: 100%;
+  background-color: #f3f6f9;
+}
+
+.header {
+  padding: 0 30px;
+  border-bottom: 1px solid #e9edf1;
+  background-color: #fff;
+}
+
+.header__nav {
+  display: flex;
+  height: 56px;
+}
+
+.header__nav-list {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.header__nav-item {
+  list-style: none;
+}
+
+.header__nav-item--popup {
+  margin-left: auto;
+}
+
+.header__nav-link {
+  display: block;
+  padding: 10px;
+  color: #757575;
+  font-weight: 600;
+  border-radius: 10px;
+  transition: ease 0.25s;
+  text-decoration: none;
+}
+
+.header__nav-link,
+.popup__link {
+  border: 2px solid transparent;
+}
+
+.header__nav-link:focus,
+.header__nav-link:focus-visible,
+.header__nav-link:hover,
+.header__nav-link:active
+.popup__link:focus,
+.popup__link:focus-visible,
+.popup__link:hover,
+.popup__link:active {
+  background-color: #F3F6F9;
+  border: 2px solid #007fff;
+  transition: ease 0.25s;
+  text-decoration: none;
+}
+
+.header__nav-link--icon {
+  width: 34px;
+  height: 34px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.header__nav-link--profile {
+  position: relative;
+  font-size: 20px;
+  text-transform: uppercase;
+  color: white;
+  background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 100 100' xmlns='http://www.w3.org/2000/svg' fill='%23757575'%3E%3Ccircle cx='50' cy='50' r='50'/%3E%3C/svg%3E");
+  background-position: center;
+  background-repeat: no-repeat;
+  z-index: 1;
+}
+
+.header__nav-link--profile:focus,
+.header__nav-link--profile:focus-visible,
+.header__nav-link--profile:hover,
+.header__nav-link--profile:active {
+  background-color: transparent;
+}
+
+.header__nav-icon {
+  width: 25px;
+  height: 25px;
+}
+
+.popup {
+  position: relative;
+}
+
+.popup__input,
```

Comment:
Watch [the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide the checkbox in the right way.